### ISSUE TITLE
[Bug Report] about command not working in DMs

### ISSUE THREAD

**Issue by Proxymiity** [jagrosh/MusicBot#530]
**Describe the bug**
`about` command not working in DM.

**To Reproduce**
1. DM the bot with `[prefix]about`

**Expected behavior**
The bot should post the about message with the version, users, etc.

**Screenshots**
No screenshots.

**Debug Info**
<details>
<summary>Debug Info</summary>

```
System Properties:
  java.version = 1.8.0_265
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 1.8
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 1.8.0_265-8u265-b01-0+deb9u1-b01
  java.specification.version = 1.8
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = 0.3.1
  Owner = 278230133176008704
  Prefix = //
  AltPrefix = ..
  MaxSeconds = 0
  NPImages = false
  SongInStatus = false
  StayInChannel = false
  UseEval = false
  UpdateAlerts = true

Dependency Information:
  JDA Version = 4.2.0_212
  JDA-Utilities Version = 3.0.4
  Lavaplayer Version = 1.3.59

Runtime Information:
  Total Memory = 738
  Used Memory = 129

Discord Information:
  ID = 512030715332919348
  Guilds = 100
  Users = 15
```

</details>

**Additional context**
The `about` command can still be executed in servers.

**Comment by Proxymiity** [comment]
I managed to get the error:
```java
# java -jar bot.jar 
[23:30:55] [ERROR] [Startup]: Could not start GUI. If you are running on a server or in a location where you cannot display a window, please run in nogui mode using the -Dnogui=true flag.
[23:30:55] [INFO] [Startup]: Loaded config from /root/musicbot.copy/config.txt
[23:30:56] [INFO] [JDA]: Login Successful!
[23:30:56] [INFO] [WebSocketClient]: Connected to WebSocket
[23:30:57] [INFO] [JDA]: Finished Loading!
[23:31:10] [ERROR] [JDA]: One of the EventListeners had an uncaught exception
java.lang.IllegalStateException: This message event did not happen in a text channel
        at net.dv8tion.jda.api.events.message.GenericMessageEvent.getTextChannel(GenericMessageEvent.java:147)
        at net.dv8tion.jda.api.events.message.GenericMessageEvent.getGuild(GenericMessageEvent.java:126)
        at com.jagrosh.jdautilities.command.CommandEvent.getGuild(CommandEvent.java:1098)
        at com.jagrosh.jdautilities.examples.command.AboutCommand.execute(AboutCommand.java:85)
        at com.jagrosh.jdautilities.command.Command.run(Command.java:323)
        at com.jagrosh.jdautilities.command.impl.CommandClientImpl.onMessageReceived(CommandClientImpl.java:557)
        at com.jagrosh.jdautilities.command.impl.CommandClientImpl.onEvent(CommandClientImpl.java:445)
        at net.dv8tion.jda.api.hooks.InterfacedEventManager.handle(InterfacedEventManager.java:96)
        at net.dv8tion.jda.internal.hooks.EventManagerProxy.handleInternally(EventManagerProxy.java:82)
        at net.dv8tion.jda.internal.hooks.EventManagerProxy.handle(EventManagerProxy.java:69)
        at net.dv8tion.jda.internal.JDAImpl.handleEvent(JDAImpl.java:147)
        at net.dv8tion.jda.internal.handle.MessageCreateHandler.handleInternally(MessageCreateHandler.java:122)
        at net.dv8tion.jda.internal.handle.SocketHandler.handle(SocketHandler.java:36)
        at net.dv8tion.jda.internal.requests.WebSocketClient.onDispatch(WebSocketClient.java:948)
        at net.dv8tion.jda.internal.requests.WebSocketClient.onEvent(WebSocketClient.java:835)
        at net.dv8tion.jda.internal.requests.WebSocketClient.handleEvent(WebSocketClient.java:813)
        at net.dv8tion.jda.internal.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:986)
        at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:385)
        at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:276)
        at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:996)
        at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:755)
        at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108)
        at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64)
        at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45)
```

**Comment by jagrosh** [comment]
Yeah, I'm aware of this and what is causing it, it's a bug in an underlying library.

**Comment by Proxymiity** [comment]
Okay. Is it the same library that causes the about command to only show 16 users instead of 300k ?

**Comment by jagrosh** [comment]
Yes, but that value is actually correct; the bot does not load all the users anymore.

**Comment by Proxymiity** [comment]
Ah I see, thanks.

**Comment by Sanduhr32** [comment]
> Yes, but that value is actually correct; the bot does not load all the users anymore.

Might want to use Guild#getMemberCount() which is provided meta data when loading a guild.

**Comment by jagrosh** [comment]
The data in the `about` command is intentionally meant to show how many users are loaded, as its a statistic about the current runtime, so the value listed is correct.

**Comment by Sanduhr32** [comment]
Could be an enhancement if enough people ask about this change, so it will say "x users loaded of n initial expected users"

**Comment by Proxymiity** [comment]
Or "x users across all servers" and "y users loaded"

**Comment by jagrosh** [comment]
This has been fixed in 0.3.2
